content/guides/getting-started-with-hugo.md

```diff
@@ -0,0 +1,178 @@
+
+ ![hugo-path-edit](images/getting-started/Hugo-Path-Edit.png "environment variables")
+
+Here’s what mine looks like after I have finished typing in where it exists (it is the one highlighted in blue). After you have done the same, you can now click ‘OK’ until all of the dialog boxes are closed. Now this is where the magic happens! After restarting your computer, try using the ‘hugo version’ command again outside of the ‘Hugo\bin’ folder and see if Windows recognizes it as a command! I simply started up my command prompt and immediately typed in ‘hugo version’ and it worked. If the command works, you are now ready to get started on a Hugo project.
+
+ ![Hugo-Version-Outside-Bin](images/getting-started/Hugo-Version-Outside-Bin.png "hugo version outside bin")
+
+
+## Mac:
+### Installing Hugo on Mac:
+The first step you will want to take is navigating to the website [Homebrew](https://brew.sh "Homebrew Installation"). This website is a package manager and allows you to install, download and manage packages. You can use [Homebrew](https://brew.sh "Homebrew Installation") to install Hugo easily on your computer. Once you are taken to the landing page, you should come across a terminal command that looks like: 
+
+```dart
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+// NOTE: 
+// This command may be updated depending on any new releases,
+// so be sure to check the website before copying the line of code above! 
+```
+
+Simply paste the line of code into your terminal. Once you click enter and input your password for administrative access, Homebrew will begin installing itself onto your machine. This can take a few minutes.	Once Homebrew is finished, you can then type the command:
+ 
+```
+brew install hugo
+```
+
+Homebrew may update itself automatically before installing hugo. 
+
+### Determining if your Installation on Windows was successful:
+
+You can validate your install by the command:
+
+```
+which hugo
+```
```

Comment:
We seem to keep choosing Themes that need Hugo extended, so let's make sure the instructions and screen shots get people that version